Assignment2_q2.py

```diff
@@ -0,0 +1,94 @@
+# coding=utf-8
+from BinaryTree import TreeNode, BinaryTree
+
+"""Finds the lowest common ancestor of two nodes in the given tree. Uses two methods:
+   first one to 'get_ancestors_recursive' to find the ancestors of each of two nodes and
+   second 'lowest_common_ancestor' to compare the ancestors of two nodes and find the lowest
+   common one.
+   Given solution takes θ(n) time, where n is the number of nodes in the tree."""
+
+
+def get_ancestors_recursive(tree, key, ancestors):
```

Comment:
This is the same function you needed in the other assignment: you can actually use it by importing it rather than copying and pasting it.

**Note:** currently the other helper function also prints the values. But that can be moved out of the helper function.